src/main/kotlin/com/openlattice/chronicle/study/StudyController.kt

```diff
@@ -104,4 +104,36 @@ class StudyController @Inject constructor(
 
     }
 
+    @Timed
+    @DeleteMapping(
+        path = [STUDY_ID_PATH],
+        produces = [MediaType.APPLICATION_JSON_VALUE],
+    )
+    override fun destroyStudy(@PathVariable studyId: UUID): Int {
+        accessCheck(AclKey(studyId), EnumSet.of(Permission.OWNER))
```

Comment:
It's not that much overhead, but `EnumSet.of(Permission.OWNER)` should be defined as a constant somewhere so that it's not recreated on every request, similar to this:

https://github.com/openlattice/openlattice/blob/develop/conductor-client/src/main/java/com/openlattice/authorization/EdmAuthorizationHelper.java#L48

Reply:
+1 Might be worth copying some of those constants into Authorizing companion object.